bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/Journal.java

```diff
@@ -390,7 +390,6 @@ public int process(boolean shouldForceWrite) throws IOException {
                     if (qe != null) {
                         cbThreadPool.execute(qe);
                     }
-                    journalStats.getJournalCbQueueSize().inc();
```

Comment:
It's possible that we actually do want to increment the `JournalCbQueueSize` here, if we're looking to measure the number of call backs being run (or waiting to run on the `cbThreadPool`). The running of this `QueueEntry` will trigger the following:

```java
        public void run() {
            if (LOG.isDebugEnabled()) {
                LOG.debug("Acknowledge Ledger: {}, Entry: {}", ledgerId, entryId);
            }
            journalCbQueueSize.dec();
            journalAddEntryStats.registerSuccessfulEvent(MathUtils.elapsedNanos(enqueueTime), TimeUnit.NANOSECONDS);
            cb.writeComplete(0, ledgerId, entryId, null, ctx);
            recycle();
        }
```

The PR currently assumes that we're measuring the number of call backs in the queue until the call back is executed and decrements the metric.